### PR TITLE
HUD function development v0.3

### LINKED ISSUE #29: Modify-Displaying-States-1
## Problem Description
Modify the positions of score, level, time, and remaining enemy count in the existing single-player mode to reduce UI differences between two-player and single-player modes.

## Todo List
- [ ] Relocate score display to the top of the screen
- [ ] Adjust level indicator position to the top of the screen
- [ ] Move time display to the top of the screen
- [ ] Reposition the remaining enemy count to the bottom center of the screen

## Feature Description
This feature aims to standardize the UI layout between single-player and two-player modes. By relocating key information (score, level, time) to the top of the screen and centralizing the remaining enemy count at the bottom, we create a more consistent user experience across different game modes.

## Additional Information
- Ensure that the new positions do not overlap with existing UI elements
- Consider the readability and visibility of the relocated elements
- Test the changes in both single-player and two-player modes to confirm consistency
- Update any related documentation or UI mockups to reflect these changes

### PR DESCRIPTION
**Pull request by DongWonLee2**
## What
This PR adds new HUD functionalities and updates. It includes the following changes:
- Modifying the Location of the Achievement Window.
- Show bomb UI on screen when bomb is acquired.
- Reconfigure UI based on score made by achievement team.

## Why
Updated for intuitive HUD screen.

## Related Issue
Closes #29 #31 #36 

## Screenshots
screenshot6

### CODE REVIEW

File: src/screen/GameScreen.java [hunk 1]
```diff
@@ -331,6 +331,8 @@ private void draw() {
 
 		// Interface.
 		drawManager.drawScore(this, this.scoreManager.getAccumulatedScore());    //clove
```
Comment (yunsful) [inline src/screen/GameScreen.java]: - The game score is being displayed multiple times.

**Review by kojesung — APPROVED** [review]
image

I have confirmed that the gitmessage.txt file has been modified and that the features implemented in the feature branch have been successfully applied.

**Review by LeeHyunWoo02 — APPROVED** [review]
image

We have confirmed that all of the corrections so far worked without any problems.

**Review by sml-logs — APPROVED** [review]
I have confirmed all functions are working well.
ScreenShot 2024-10-10 오후 5 46 45

**Review by yunsful — APPROVED** [review]
스크린샷 2024-10-10 오후 5 42 05

LGTM

**Review by aeioiie — APPROVED** [review]
image
I have confirmed everything working well. Thanks for your effort!

**Review by min5421 — APPROVED** [review]
캡처
 
All functions work well!

**Pull request merged**